pkg/crt/versionfile.go

```diff
@@ -22,7 +22,7 @@ func getCoreVersion(dir string) (*version.Version, error) {
 	return getCoreVersionFromVersionFile(dir)
 }
 
-var versionSearchPath = []string{".", ".release", "dev"}
+var versionSearchPath = []string{".", ".release", "version", "dev"}
```

Comment:
Bonus addition.. I saw in another PR that we're now looking for `version/VERSION` as the path for the version file, so now this action supports the same path.